services/QuillLMS/app/workers/test_for_earned_checkboxes_worker.rb

```diff
@@ -4,14 +4,14 @@ class TestForEarnedCheckboxesWorker
 
   def perform(id)
     teacher = User.find_by(id: id)
-    if teacher
-      #we don't want to trigger analtyics since this is used as a callback after login
-      find_or_create_checkbox('Create a Classroom', teacher) if teacher.classrooms_i_own.any?
-      find_or_create_checkbox('Add Students', teacher) if teacher.classrooms_i_own.find{|classroom| classroom.students.any?}
-      #finds all types of assigned units and ensures they have checkboxes
-      assigned_unit_types = teacher.units.map(&:unit_activities).flatten.map(&:checkbox_type).uniq.each{|type| find_or_create_checkbox(type, teacher)}
-      find_or_create_checkbox('Add School', teacher) if teacher.school.present?
-    end
+    return unless teacher
```

Comment:
Super duper approve of using the early return here over the `if`.

Another Ruby syntax thing that might be useful in the future: the return value of the assignment operator (`=`) is the value that the variable on the left is set to, so you can actually do:
```ruby
return unless teacher = User.find(id)
```
(`find()` always tries to find by `id`)

Again, not worth bothering with for this PR.